_data/hosting_providers.json

```diff
@@ -1360,13 +1360,13 @@
     "note": ""
   },
   {
-    "name": "OVH.com",
-    "link": "",
-    "category": "partial",
-    "tutorial": "",
+    "name": "OVHcloud",
+    "link": "https://www.ovh.ie/web-hosting/",
+    "category": "full",
+    "tutorial": "https://docs.ovh.com/gb/en/hosting/activate-https-website-ssl/",
```

Comment:
This tutorial describes a set of manual steps you have to take to activate HTTPS on your site when using OVHcloud, but links like https://www.ovh.ie/web-hosting/shared-ssl.xml talk about how this is set up by default. Do you know how this works?

Reply:
New hostings have SSL enabled by default with the associated domain.
When creating additional subdomains (aka multi-site) you have to check the option, I'm going to see with the team to have it enabled by default.
The tutorial link shows the whole procedure to force https redirection on your website including LE setup for websites that didn't chose to activate it initally.

Reply:
Thanks for the explanation!

I agree with you checking with the team about enabling TLS by default for additional subdomains is a good idea, but based on the current state of things, I think you filled out everything here correctly so I'm merging this PR.

Thanks again.